mta/fixtures/tags.py

```diff
@@ -2,15 +2,15 @@
 
 
 @pytest.fixture()
-def tag_types_names(get_api):
+def tagcategories_names(get_api):
     # Note: tag type name is unique
-    return {tag_type.name for tag_type in get_api.tagtypes_get()}
+    return {tag_category.name for tag_category in get_api.tagcategories_get()}
```

Comment:
let's raname this to tagCategory.name
and 
get_api.tagCategories()

Reply:
get_api.tagcategories_get() is a function from swagger_client
AFAIK, the naming conventions in Pyhon are to use lower cases, words separated by underscores for variables, functions and file name. Upper case for the first letter of each word in Class name only (no underscores)

Reply:
Ok we can follow the python conventions . 

get_api.tagcategories is much more clear and more readable than 
get_api.tagcategories_get()

But I will leave it to your decision.

Reply:
This function was generated from the swagger.yaml file, we should not change its name.
If we do so, the test will fail next time the swagger_client will be generated